src/utils/validationSchemas/authenticationSchemas.js

```diff
@@ -22,11 +22,26 @@ const SignUpSchema = Joi.object().keys({
     .pattern(
       /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0123456789])(?=.*[@$!%*?&])[A-Za-z0123456789@$!%*?&]{8,}$/
     )
-    .messages(errorMessage('Password')), // password has both numbers and letters and is btn 6 and 30
+    .messages(errorMessage('Password')),
 });
 
 const CollectionNameSchema = Joi.object().keys({
   name: Joi.string().required().messages(errorMessage('Collection Name')),
 });
 
-export { LoginSchema, SignUpSchema, CollectionNameSchema };
+const PasswordSchema = Joi.object().keys({
+  oldPassword: Joi.string()
+    .required()
+    .pattern(
+      /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0123456789])(?=.*[@$!%*?&])[A-Za-z0123456789@$!%*?&]{8,}$/
+    )
+    .messages(errorMessage('Password')),
+  newPassword: Joi.string()
+    .required()
+    .pattern(
+      /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0123456789])(?=.*[@$!%*?&])[A-Za-z0123456789@$!%*?&]{8,}$/
+    )
+    .messages(errorMessage('Password')),
+});
```

Comment:
check that they are not equal here so that you can remove the conditional check middleware

Reply:
Thanks, it's done.